homeassistant/components/kodi/browse_media.py

```diff
@@ -280,6 +286,11 @@ def item_payload(item, media_library):
         thumbnail=thumbnail,
     )
 
+    if "tvshowid" in item or "movieid" in item:
+        payload.sortable_title = sortable_title
```

Comment:
This adds a new attribute to the `BrowseMedia` instance that is not defined in the init method of the instance. That's not really good practice.

Reply:
Oh absolutely, we can. Originally I was adding it to `BrowseMedia` to make it more flexible for integrations with the other players.

What do you think of this instead? In line 180 I could do this:
```python
    if search_type in (MEDIA_TYPE_TVSHOW, MEDIA_TYPE_MOVIE):
        children.sort(key=lambda x: x.title.replace("The ", "", 1), reverse=False)
```

Reply:
Yeah, doing it like that is better.

Reply:
Sounds good! Thanks a lot for the feedback @balloob, @MartinHjelmare.